ewcmd.py

```diff
@@ -817,6 +817,61 @@ def weather_txt(id_server):
 
 	await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 
+async def pray(cmd):
+	user_data = EwUser(member = cmd.message.author)
+	resp_cont = ewutils.EwResponseContainer(id_server = user_data.id_server)
+
+	if user_data.poi != ewcfg.poi_id_endlesswar:
+		response = "You must be in the presence of your lord if you wish to pray to him."
+
+	else:
+		# Generates a random integer from 1 to 100. If it is below the prob of poudrin, the player gets a poudrin.
+		# If the random integer is above prob of poudrin but below probofpoud+probofdeath, then the player dies. Else,
+		# the player is blessed with a response from EW.
+		probabilityofpoudrin = 10
+		probabilityofdeath = 10
+		diceroll = random.randint(1, 100)
+
+		if diceroll < probabilityofpoudrin: # Player gets a poudrin.
+			item = random.choice(ewcfg.mine_results)
+
+			item_props = ewitem.gen_item_props(item)
+
+			ewitem.item_create(
+				item_type = item.item_type,
+				id_user = cmd.message.author.id,
+				id_server = cmd.message.server.id,
+				item_props = item_props
+			)
+
+			response = "ENDLESS WAR takes pity on you, and with a minor tremor he materializes a {} in your pocket.".format(item.str_name)
+
+		elif diceroll < (probabilityofpoudrin + probabilityofdeath): # Player gets a face full of bone-hurting beam.
+			response = "ENDLESS WAR doesn’t respond. You squint, looking directly into his eye, and think you begin to see particle effects begin to accumulate..."
+			await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
+			await asyncio.sleep(3)
+
+			user_data.die(cause = ewcfg.cause_praying)
```

Comment:
we need to reload a player's data after awaiting a command. otherwise it may overwrite changes that were made in the meantime